deps.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# Call this script to install test dependencies for all subpackages.
+
+set -eux
+
+# We should install rock dependencies to corresponding directory.
+# If you run ./queue/deps.sh instead of cd ./queue; ./deps.sh,
```

Comment:
I think you can solve it with commands grouping, see https://www.gnu.org/software/bash/manual/html_node/Command-Grouping.html.

In short: There is just one single difference between command grouping using “{ }” and “( )“:
- { Commands; } : Commands execute in current shell
- ( Commands ) : Commands will execute in a subshell


`deps.sh` that wraps another `deps.sh` looks weird.

Reply:
used subshell approach